### PR TITLE
MGMT-12026: Fix e2e-metal-assisted-prod

### PR DESCRIPTION
**Pull request by adriengentil**

### CODE REVIEW

**Comment by adriengentil** [comment]
Build succeeded here: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/32231/rehearse-32231-pull-ci-openshift-assisted-test-infra-master-e2e-metal-assisted-prod/1569679043819737088
(the build it failed because I pushed before the gather step finished :man_facepalming: but the `common-test` step succeeded)

File: ci-operator/step-registry/assisted/common/setup/prepare/assisted-common-setup-prepare-commands.sh [hunk 1]
```diff
@@ -96,7 +96,9 @@ export PULL_SECRET=\$(cat /root/prod/pull-secret)
 export OFFLINE_TOKEN=\$(cat /root/prod/offline-token)
 export REMOTE_SERVICE_URL=https://api.openshift.com
 export NO_MINIKUBE=true
-export MAKEFILE_TARGET='setup test_parallel'
+export MAKEFILE_SETUP_TARGET=setup
+export MAKEFILE_TARGET=test_parallel
+export WORKER_DISK={{ '120 GB' | human_to_bytes }}
```
Comment (adriengentil) [inline ci-operator/step-registry/assisted/common/setup/prepare/assisted-common-setup-prepare-commands.sh:101]: I just replicated what was done in the old script but I don;t know if it's still required

Comment (osherdp) [reply ci-operator/step-registry/assisted/common/setup/prepare/assisted-common-setup-prepare-commands.sh:101]: test-infra should have handled it: https://github.com/openshift/assisted-test-infra/blob/c8c6bdbf2b2c3be1d4ee6fc1bcd96dcb466d30af/src/triggers/default_triggers.py#L12

**Review by osherdp — APPROVED** [review]

**Comment by osherdp** [comment]
/hold
ah wait, it should be 100GB

File: ci-operator/step-registry/assisted/common/setup/prepare/assisted-common-setup-prepare-commands.sh (hunk 1 above)
Comment (adriengentil) [reply ci-operator/step-registry/assisted/common/setup/prepare/assisted-common-setup-prepare-commands.sh:101]: let's not set it then ?

Comment (osherdp) [reply ci-operator/step-registry/assisted/common/setup/prepare/assisted-common-setup-prepare-commands.sh]: yes let's try without setting it

**Review by osherdp — APPROVED** [review]

**Pull request merged**